### DISCUSSION TITLE
Only plot Legend / Plot Legend seperately

### DISCUSSION THREAD

**Discussion by mobergmann** [JuliaPlots/Makie.jl#4598] [Q&A] [answered]
I am currently trying to save the legend of a plot separately, because many of my plots use the same legend and I only want to include them in my latex document once, and that time without a plot.

Is it possible to only plot the legend, and when yes, how? I tried to research it the whole day long, but couldn't find a solution.  
Thanks in advance for your help.

---

BTW, this one of my plots:
```julia
using CairoMakie

...

fig = Figure()
ax = Axis(
    fig[1, 1],
    xlabel = "Time (minutes)",
    ylabel = "Moisture",
    aspect = AxisAspect(1)
)

for (i,pad) in enumerate(eachrow(y_pads))
    lines!(ax, x, pad, label="Pad $i")
end

# add all time pouring
for i in eachindex(vlines_max)
    band!(ax, [vlines_min[i], vlines_max[i]], [0,0], [1100,1100], color=(:gray, 0.5), label="Pouring")
end

fig[1, 2] = Legend(fig, ax, "Legend", merge=true, nbanks=2)

save("$(@__DIR__)/plots/moisture.pdf", fig)
```

**Comment by jkrumbiegel** [comment, marked as answer]
This should work I think, a legend does not need to be in the same figure as the content it gets its entries populated with:

```
fig1 = ...
stuff_for_legend = plot_things(fig1,...)

fig2 = Figure()
Legend(fig2[1,1], stuff_for_legend, some_labels, tellheight = true)
resize_to_layout!(fig2)
save("legend.svg", fig2)
```

**Reply by mobergmann** [reply]
That did the trick. Many thanks!
